evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -548,32 +548,23 @@ def test_indexing(X_y_binary, logistic_regression_binary_pipeline_class):
         clf[:1]
 
 
-def test_describe(caplog, logistic_regression_binary_pipeline_class):
```

Comment:
Combined `test_describe` and `test_describe_nonlinear` and `test_describe_fitted` and `test_describe_nonlinear_fitted`

Reply:
Nice!